plugins/org.eclipse.osee.framework.ui.skynet/src/org/eclipse/osee/framework/ui/skynet/widgets/XTextDam.java

```diff
@@ -182,15 +184,31 @@ public void setEditorData(EditorData editorData) {
    @Override
    public void validate() {
       if (Widgets.isAccessible(sText)) {
+         List<Boolean> enabledAll = new ArrayList<Boolean>();
          for (ConditionalRule rule : getConditions()) {
+            boolean enabled = false;
```

Comment:
would seem that this check belongs in ConditionalRule and not in XTextDam?  And validate in XWidget and not XTextDam?  This has to work for all widgets, not just text/string?

Reply:
Done